cms/templates/settings.html

```diff
@@ -251,10 +253,45 @@ <h2 class="title-2">${_('Course Schedule')}</h2>
           % if can_show_certificate_available_date_field(context_course):
           <ol class="list-input">
             <li class="field-group field-group-certificate-available" id="certificate-available">
-              <div class="field date" id="field-certificate-available-date">
+              <div class="field date" id="field-certificates-display-behavior">
+                <label for="certificates-display-behavior">${_("Certificates Display Behavior")}</label>
+                <select id="certificates-display-behavior">
+                  <option value="early_no_info">Immediately upon passing</option>
```

Comment:
Should these option strings also be translated?

Reply:
Good call. Updating